### PR TITLE
LayerStyler changes for internal Bayer use

### PR DESCRIPTION
**Pull request by drodenberg**
### PR Safety Checklist:

 - [x] Added the task to the appropriate release doc under **Enhancements** or **Bug Fixes**
 - [x] Bump `package.json` & `package-lock.json` version numbers to appropriate release
 - [x] (optional) All external API changes have been documented
 - [x] (optional) Build docs: `npm run docs`

### CODE REVIEW

File: src/LayerStyler/LayerStyler.js [hunk 1]
```diff
@@ -43,6 +44,8 @@ class LayerStyler extends React.Component {
     // make sure the attributes get updated each time the view extent changes
     const listeners = addMovementListener(map, () => this.forceUpdate())
 
+    this.props.onComponentMount()
```
Comment (drodenberg) [inline src/LayerStyler/LayerStyler.js:47]: remove this

File: src/LayerStyler/LayerStyler.js [hunk 2]
```diff
@@ -204,12 +207,21 @@ class LayerStyler extends React.Component {
...
+
 LayerStyler.propTypes = {
   /** Openlayers map object */
   map: PropTypes.object.isRequired,
 
   /** Object with key/value pairs for translated strings */
-  translations: PropTypes.object.isRequired
+  translations: PropTypes.object.isRequired,
+
+  whitelistedLayers: PropTypes.array,
```
Comment (drodenberg) [inline src/LayerStyler/LayerStyler.js:222]: comments

File: src/LayerStyler/LayerStyler.js [hunk 3]
```diff
@@ -204,12 +207,21 @@ class LayerStyler extends React.Component {
...
+
+  onComponentMount: PropTypes.func
```
Comment (drodenberg) [inline src/LayerStyler/LayerStyler.js:224]: removed

File: src/LayerStyler/StyleManager/index.js [hunk 4]
```diff
@@ -166,7 +167,14 @@ class StyleManager extends Component {
...
     const layerTitles = layers.map(getTitleForLayer)
-    const layerSelected = activeIdx !== null
+    const layerSelected = activeIdx !== null // eslint-disable-line
+    let tooltipMsg = ''
+
+    if (!layerSelected) {
+      tooltipMsg = 'Select a Geoserver Layer to filter attributes'
+    } else if (!layers[activeIdx].isGeoserverLayer) {
+      tooltipMsg = 'VectorLayers dont have filter capabilities, only Geoserver Layers do'
+    }
```
Comment (drodenberg) [inline src/LayerStyler/StyleManager/index.js:177]: move to translations

Unsupported layer type for filtering Layer attributes (Supported types include GeoserverLayer)

File: src/LayerStyler/StyleManager/styled.js [hunk 5]
```diff
@@ -11,8 +15,13 @@ export const InputContainer = styled.div`
...
-  padding: 36px 0;
   text-align: center;
   vertical-align: top;
 `
+
+export const Tooltip = withStyles(() => ({
+  tooltip: {
+    fontSize: '15px',
+    paddingTop: '-36px'
+  }
```
Comment (drodenberg) [inline src/LayerStyler/StyleManager/styled.js:26]: remove padding top

**Review by stazrad — APPROVED** [review]

**Pull request merged**